Assets/Scripts/CameraEffects.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using UnityEngine;
+using System.Threading.Tasks;
+
+public class CameraEffects : MonoBehaviour
+{
+  private static float magnitude;
+
```

Comment:
Also add `[Header("MAGNITUDE")]` as a single line before these attributes (note: there is no semicolon after this line). `Header` creates a label for your variables in the editor. Just a nice little way to stay organized!

Reply:
When doing this, does it expose the variable to the editor?

Reply:
The header line doesn't expose the variable to the editor, but declaring a variable as public will. Alternatively, Mike and I were talking about changing the variables we want to expose to `[SerializeField] private` instead of `public`. This way, the variables will appear in the editor, but still be private to other classes.

So let's do that instead - use `[SerializeField] private` to expose the variable to the editor so we can make changes to the magnitudes through the editor instead of code.